client/src/main/java/com/aapeli/tools/Tools.java

```diff
@@ -113,15 +113,15 @@ public static int getRandomByPossibility(double[] var0) {
         }
     }
 
-    public static <E> String[] vectorToStringArray(Vector<E> var0) {
+    public static <E> String[] listToStringArray(List<E> var0) {
```

Comment:
We can simplify this entire method with `list.stream().map(Object::toString)toArray(String[]::new);`